CorsixTH/Lua/world.lua

```diff
@@ -1020,13 +1020,13 @@ function World:onTick()
         self.day = 1
         self.game_date = self.game_date:plusMonths(1)
         if self.game_date:monthOfYear() > 12 then
```

Comment:
This condition will never be true now.

Reply:
Yep thanks. I am aware of this and the next commit should fix that.